proto/panacea/datapool/v2/pool.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+package panacea.datapool.v2;
+
+option go_package = "github.com/medibloc/panacea-core/x/datapool/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "google/protobuf/duration.proto";
+
+// Pool defines a data pool
+message Pool {
+  uint64                    pool_id = 1;
+  uint64                    round = 2;
+  repeated string           data_schema = 3;
+  uint64                    target_num_data = 4; // target number of data
+  uint64                    cur_num_data = 5;    // current number of data
+  uint64                    max_nft_supply = 6;  // max supply of data access NFT of the pool
+  uint64                    issued_nft = 7;      // current issued NFT
```

Comment:
is this `issued_nft` means the curation NFT ID?

Reply:
The latter. It would be better to use `num_issued_nft` to make clearer. Thanks.